packages/subscription-manager/src/components/index.d.ts

```diff
@@ -89,3 +89,23 @@ declare module 'calypso/components/route' {
 		currentQuery: unknown;
 	};
 }
+
+declare module 'calypso/components/forms/form-checkbox' {
+	const FormInputCheckbox: React.FC< InputHTMLAttributes< HTMLInputElement > >;
+	export default FormInputCheckbox;
+}
+
+declare module 'calypso/components/forms/form-fieldset' {
+	const FormFieldset: React.FC< InputHTMLAttributes< HTMLInputElement > >;
+	export default FormFieldset;
+}
+
+declare module 'calypso/components/forms/form-label' {
+	interface Props {
+		optional?: boolean;
+		required?: boolean;
+	}
+	type LabelProps = LabelHTMLAttributes< HTMLLabelElement >;
+	const FormLabel: FunctionComponent< Props & LabelProps >;
```

Comment:
Small one: Everywhere else in the file, we use React.FC

Reply:
Nice. Will change 👍